modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -909,6 +923,14 @@ void ONNXCompiled::Run(const std::vector<cv::Mat>& ins,
                          out_run_names.data(),
                          &out_tensors.front(),
                          params.output_names.size());
+        if (out_tensor_info[0].type == ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64) { 
```

Comment:
some `NB` comment would be useful here to understand why it's needed

Reply:
done